src/layouts/Field/Field.js

```diff
@@ -35,7 +35,10 @@ class Field extends React.Component {
      * (stretch|left|right): the alignment of contents within the Field
      */
     alignContent: PropTypes.oneOf(['stretch', 'left', 'right']),
-
+    /**
+     *  (top|center|bottom): the horizon alignment of content within the Field
```

Comment:
sp: horizontal

Reply:
And if it's `top|center|bottom`, wouldn't that be vertical alignment?

Reply:
khm, I thought it calls `horizontal` OK will change it, thank you!